tools/sklearn/discriminant.xml

```diff
@@ -106,7 +106,7 @@ with open("$outfile_fit", 'wb') as out_handler:
             <param name="selected_algorithm" value="LinearDiscriminantAnalysis"/>
             <param name="solver" value="svd" />
             <param name="store_covariance" value="True"/>
-            <output name="outfile_fit" file="lda_model01" compare="sim_size" delta="500"/>
+            <output name="outfile_fit" file="lda_model01" compare="sim_size" delta="1"/>
```

Comment:
The value of `delta` varies throughout. Some places it is `1`, other places `5`. Is it possible to use just one value?

Reply:
The file size varies a lot. For MB sized file, dalta 500 is good enough, but for a file of hundreds of Byte, delta 1 gives better accuracy.

Reply:
I am not sure I understood it completely. Here `sim_size` is being used under tests. How it is related to the performance of an algorithm?

Reply:
Sorry. The accuracy doesn't relate to performance, but points to the right output. The delta, in my understanding, is the difference between the output size and the test data file size. If the size of output is expected to be in hundreds of Bytes, delta 500 may provides little assurance that the output from the test is the same as the test data file.